jenkins/databricks/build.sh

```diff
@@ -107,7 +107,8 @@ sudo cp $CUDF_JAR $DB_JAR_LOC
 
```

Comment:
2021 copyrights need to be added to this file

Reply:
fixed